test/integration/regressions/regression-893-897-test.coffee

```diff
@@ -0,0 +1,94 @@
+{assert} = require('chai')
+
+Dredd = require('../../../src/dredd')
+{runDreddWithServer, createServer} = require('../helpers')
+
+
+describe('Regression: Issue #893 and #897', ->
+  describe('when the response has no explicit status code', ->
+    runtimeInfo = undefined
+
+    beforeEach((done) ->
```

Comment:
I don't think it is necessary to use `beforeEach` here (use `before`), the server is the same for each assertion.